screens/map/StoreDetailsScreen.js

```diff
@@ -3,7 +3,6 @@ import { Linking } from 'expo';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { ScrollView, TouchableOpacity, View } from 'react-native';
-import { Chip } from 'react-native-paper';
```

Comment:
goodbye chippy :')

Reply:
it's ok he just moved files : ' ) he's in a better place now :'   )